packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -57,7 +69,7 @@ When customers [integrate Okta with an OpenID Connect-based Identity Provider](/
 
 * When ThreatInsight evaluated sign-in attempts for unknown users, the threat level was incorrectly displayed as `threatLevel=UNKNOWN` in the System Log. (OKTA-471299)
 
-* The [OAuth token endpoint](/docs/reference/api/oidc/#token) accepted requests that included the `resource` parameter. (OKTA-476549)
+* The OAuth 2.0 [`/token`](/docs/reference/api/oidc/#token) and [`/authorize`](/docs/reference/api/oidc/#authorize) endpoints accepted requests that included the `resource` parameter. (OKTA-476549)
```

Comment:
I think that the SMEs said that this bug wasn't an issue on the `authorize` endpoint, but that they included the fix on `authorize` to be safe. Does the bug apply in this case?

Reply:
The Dev and Dev Mgr confirmed to add /authorize in the bug blurb. Thanks